Documentation/CRDs/Cluster/external-cluster/external-cluster.md

```diff
@@ -275,3 +256,33 @@ you can export the settings from this cluster with the following steps.
 
 !!! important
     For other clusters to connect to storage in this cluster, Rook must be configured with a networking configuration that is accessible from other clusters. Most commonly this is done by enabling host networking in the CephCluster CR so the Ceph daemons will be addressable by their host IPs.
+
+## Upgrades
+
+1. If consumer cluster doesn't have restricted caps, this will upgrade all the default csi-users (non-restricted):
```

Comment:
Should this begin as a sub-section like `### Upgrade example - CSI capabilities`?

Reply:
No need for subheading as it will always be needed